wpilibc/src/main/native/cpp/Drive/MecanumDrive.cpp

```diff
@@ -77,9 +77,9 @@ void MecanumDrive::DriveCartesian(double ySpeed, double xSpeed,
 
   double wheelSpeeds[4];
   wheelSpeeds[kFrontLeft] = input.x + input.y + zRotation;
-  wheelSpeeds[kFrontRight] = input.x - input.y + zRotation;
+  wheelSpeeds[kFrontRight] = input.x + input.y * m_rightSideInvertMultiplier + zRotation;
   wheelSpeeds[kRearLeft] = -input.x + input.y + zRotation;
-  wheelSpeeds[kRearRight] = -input.x - input.y + zRotation;
+  wheelSpeeds[kRearRight] = -input.x + input.y * m_rightSideInvertMultiplier + zRotation;
```

Comment:
Is the math here correct?  Just want to make sure that we shouldn't be doing something with input.x as well.

Reply:
I think so. @calcmogul what do you think?